readme.md

```diff
@@ -40,7 +40,130 @@ TBA
 
 ### 💽 DB Schema
 
-![db schema](https://github-production-user-asset-6210df.s3.amazonaws.com/19575839/261530664-6140d0e8-abf8-4fb8-80c9-ff9eb7adfcb6.png)
+```mermaid
+
+erDiagram
```

Comment:
please remove tables that were not implemented yet

Reply:
done